### PR TITLE
Pixel starting up

### PR DESCRIPTION
**Pull request by Xuan-Wang-Summer**

### CODE REVIEW

**Comment by Anvesha4** [comment]
I reviewed your changes and left some comments on pixel workshop. Please take a look and reach out to me if you have questions! Also, please pull latest changes from UCSD_Interns branch into your branch to remove unintended changes from the PR.

File: content/english/python-pixel/starting-up/conditionals-and-loops/_index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "If-else Condition"
```
[the remark this answers is not in the document]
Comment (Xuan-Wang-Summer) [reply content/english/python-pixel/starting-up/conditionals-and-loops/_index.md:2]: But we need to talk about "the else-if condition" and some other new python concepts, which we will highly rely on in order to elaborate pixel concept. I am not sure whether we should add them to other workshops, which may their workshop look very different from the original version, or we should do a review in pixel workshop.

File: content/english/python-pixel/starting-up/_index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Python: Starting up"
+icon: "fab fa-python"
+draft: false
+weight: 10
+---
+In this section, we will review some basic knowledge needed to learn pixels in Python, if you already knew those basics, you can skip these sections.
```
[the remark this answers is not in the document]
Comment (Xuan-Wang-Summer) [reply content/english/python-pixel/starting-up/_index.md:7]: Sure, we will do that.

File: content/english/python-pixel/_index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Python: Pixels"
+description: "Pixels and Colors in Python"
+prereq: "Python Basics"
+difficulty: "Progressor"
```
[the remark this answers is not in the document]
Comment (Xuan-Wang-Summer) [reply content/english/python-pixel/_index.md:5]: Sure, I am going to change it to "Intermediate" and will submit a new pull request soon!

**Pull request closed without merging**